dask_cloudprovider/providers/aws/ecs.py

```diff
@@ -948,8 +998,17 @@ def _task_role_name(self):
                     "cpu": self._worker_cpu,
                     "memory": self._worker_mem,
                     "memoryReservation": self._worker_mem,
+                    "resourceRequirements": resource_requirements,
                     "essential": True,
-                    "command": ["dask-worker"],
+                    "command": [
+                        "dask-cuda-worker" if self._worker_gpu else "dask-worker",
+                        "--nthreads",
+                        "{}".format(int(self._worker_cpu / 1024)),
+                        "--memory-limit",
+                        "{}GB".format(int(self._worker_mem / 1024)),
```

Comment:
You can also use `MB` or `MiB` here if that's more convenient

Reply:
These values are used in multiple places and those places are less flexible 😄

Reply:
I'm not suggesting that we use different units for `self._worker_mem`

Reply:
```suggestion
                        "{}MB".format(int(self._worker_mem)),
```

Reply:
Ohh I see 😄